rest/model/src/main/java/io/syndesis/model/Violation.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.syndesis.rest.v1.operations;
+package io.syndesis.model;
```

Comment:
what's the difference between "message" and "error" in Violation ?

Reply:
in fromConstraintViolation() is this all null safe ? (sorry can not comment directly on the code).

Reply:
The `message` holds the text of an error (`"XYZ is required"`), and error contains the class of error (`"NotEmpty"`).

Should be null safe, it's the way bean validation works, constraints are applied via annotations, and each violation has a resolved message (or a placeholder message), if that's what you mean.